src/main/webapp/consentGroup/NewConsentGroup.js

```diff
@@ -537,7 +479,7 @@ class NewConsentGroup extends Component {
     }).catch(error => {
       console.error(error);
     });
-  };
+  }
```

Comment:
I see a lot of this kind of churn where one person on the team adds semi-colons and others remove them. Can we look into a standard? Maybe a lint file that enforces style would be helpful.

Reply:
sure!